src/index.ts

```diff
@@ -1,4 +1,6 @@
 import './global.scss';
+import './styles/main.scss';
+import './styles/authorization.scss';
```

Comment:
we need to place all scss file in style.scss in folder styles, and add in index.ts only one import style.scss